### PR TITLE
Reworked Gutenberg integration of ISC

### PR DESCRIPTION
**Pull request by webzunft**

### CODE REVIEW

File: gutenberg/gutenberg.php [hunk 1]
```diff
@@ -1,101 +1,75 @@
...
+		wp_register_script(
+			'isc/image-block',
+			plugin_dir_url( __FILE__ ) . 'isc-image-block.js',
+			array( 'jquery', 'wp-api', 'lodash', 'wp-blocks', 'wp-editor', 'wp-element', 'wp-i18n', ),
+			true
+		);
+		
+		global $wpdb;
+		$table = $wpdb->prefix . 'postmeta';
+		$query = "SELECT * FROM $table WHERE `meta_key` LIKE '%isc_image%'";
```
Comment (webzunft) [inline gutenberg/gutenberg.php:44]: Please use an escape function

File: gutenberg/gutenberg.php [hunk 2]
```diff
@@ -1,101 +1,75 @@
...
+			if ( current_user_can( 'edit_posts' ) ) {
+				update_post_meta( absint( $_post['id'] ), $_post['key'], $_post['value'] );
+				wp_send_json( $_post );
+			}
+		}
+		die;
+	}
+	
+	/**
+	 * Enqueue JS file
```
Comment (webzunft) [inline gutenberg/gutenberg.php:31]: This function seems to do more than just enqueuing JS files. Please leave more information in the description and as inline comments

File: gutenberg/gutenberg.php (hunk 1 above)
Comment (cnhk) [reply gutenberg/gutenberg.php]: https://github.com/cnhk/image-source-control/commit/edf48b25d8aa3a25c54df840da03d3a2b06afc81

File: gutenberg/gutenberg.php (hunk 2 above)
Comment (cnhk) [reply gutenberg/gutenberg.php]: https://github.com/cnhk/image-source-control/commit/edf48b25d8aa3a25c54df840da03d3a2b06afc81

**Comment by webzunft** [comment]
@cnhk I found a few bugs I didn’t noticed before.

The _this is my image_ option does not react when I disable it
The _URL_ option cannot be changed correctly.

See the video on https://www.dropbox.com/s/mcdm3s6unabxayx/200502-rija-isc-bugs.mp4?dl=0

**Comment by cnhk** [comment]
@webzunft Just fixed the two bugs you described in the video

**Pull request merged**